actioncable/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+*   Add PostgreSQL subscription connection identificator
+
+    Now you can distinguish ActionCable's PostgreSQL subscription connections among others.
+    Also, you can set custom `id` in `cable.yml` configuration.
```

Comment:
`id` sounds confusing: what does it identify?
I think, we need something more descriptive.

Maybe, `adapter_process_id`?

Reply:
@palkan This is not something I want to change because it was introduced in #33798 and can potentially break somebody's expectations.